BLSQ/iaso
# Polio 1207 more and more reasons for delay

Add 2 more reasons for delay

Related JIRA tickets :POLIO-1207

## Self proofreading checklist


## Changes

- Add 2 reasons for delay in model + migration
- Update front-end: 
    - Add translations
    - update reasons list 

## How to test

Go to polio>Campaigns>Rounds
Update dates for a round: the new reasons should be visible (bottom 2)
Try updating and saving the round

## Print screen / video

https://github.com/BLSQ/iaso/assets/38907762/52b85887-8ec7-4d01-bd7d-b1e8ac9af9ce




## Notes

Merges into hotfix-reasons-for-delay-VRF
